ptf/testplan.md

```diff
@@ -232,6 +232,12 @@ SAI PTF TESTPLAN contains all test cases coverd in ptf/ directory divided by fun
 
 ## Hostif
 
+## Isolation Group
+| Test case id | Description | Test name |
+| ------------ | ----------- | --------- |
+| isolationgroup.1 | Verify isolation group CRUD operations | saiisolationgroup.PortIsolationTest.attributeTest |
+| isolationgroup.2 | Test forwarding between ports with isolation groups attached | saiisolationgroup.PortIsolationTest.forwardingTest |
```

Comment:
I'm not sure about those tests. They seem HW specific, I couldn't run them on model on x1_tofino and x4_tofino profiles. 

Forwarding test specifies:
        Test forwarding between ports with isolation groups attached
        Ingress ports: port10 and port24
        Isolation groups:
          grp1: port 11 and port12
          grp2: port 12 and port13